.github/CODEOWNERS

```diff
@@ -61,3 +61,10 @@
  
 # ServiceLabel: %area-ServiceBus
 # ServiceOwners:                       @shankarsama @EldertGrootenboer
+
+# PRLabel: %area-BestPractices
+/src/Commands/AzureBestPractices/           @g2vinay @conniey 
+/src/Resources/azure-best-practices.txt     @g2vinay @conniey
+
+# ServiceLabel: %area-BestPractices
+# ServiceOwners: 
```

Comment:
there should be at least one owner here, or else the `Service Attention Required` engsys infra will fail.  Even if that contact should be you.

Reply:
added myself and connie